lib/howl/application.moon

```diff
@@ -116,6 +126,18 @@ class Application extends PropertyObject
 
     @_buffers = [b for b in *@_buffers when b != buffer]
 
+    if buffer.file
+      @_recently_closed = [file_info for file_info in *@_recently_closed when file_info.file != buffer.file]
```

Comment:
This block could be its own private method instead, as to not clutter `close_buffer` to much.